drivers/watchdog/wdt_ite_it8xxx2.c

```diff
@@ -211,12 +213,13 @@ static void wdt_it8xxx2_isr(const struct device *dev)
 		data->callback(dev, 0);
 	}
 
+#ifdef CONFIG_WDT_ITE_REDUCE_WARNING_LEADING_TIME
```

Comment:
Can we use `if (IS_ENABLED(...)) {` instead?

Reply:
I will create a PR to cleanup this and kscan together.